shovel/utils/setup_indices.py

```diff
@@ -15,8 +15,8 @@ def setup_indices():
             "image": {
                 "properties": {
                     "tags": {"type": "text"},
-                    "file_url": {"type": "text"},
-                    "team_id": {"type": "text"},
+                    "file_url": {"type": "keyword"},
+                    "team_id": {"type": "keyword"},
```

Comment:
What's up with that?

Reply:
basically the mappings are elasticsearches "Schema" so team_id had the type text instead of keyword.
Text is good because it supports full text search with partial matchin etc. but it doesn't support filtering, sorting and other more "aggregational functions". So for IDs, emails, etc. you should use the type keyword instead of text. We aren't currently using the file_url for anything but if we were I think it should be a keyword as well.

Also btw this was already done inside the migrations folder so if you ran setup indices + migrate everything would've already worked. But why keep the old schema here? doesn't make any sense to me.

Reply:
Ok thanks for the explanation!